mobile-app/app/contexts/__mocks__/FeatureFlagContext.tsx

```diff
@@ -13,6 +13,14 @@ export function useFeatureFlagContext (): FeatureFlagContextI {
         description: 'Browse loan tokens provided by DeFiChain',
         networks: [EnvironmentNetwork.LocalPlayground, EnvironmentNetwork.RemotePlayground],
         platforms: ['ios', 'android', 'web']
+      }, {
+        id: 'auction',
+        name: 'Auction',
+        stage: 'beta',
```

Comment:
Maybe alpha stage only since API is not yet ready

Reply:
This is mock for feature flags context

Reply:
ha my bad